modules/installation-alibaba-dns.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-account.adoc
+
+:_content-type: PROCEDURE
+[id="installation-alibaba-dns_{context}"]
+= Registering and Configuring Alibaba Cloud Domain
+
+To install {product-title}, the Alibaba Cloud account you use must
```

Comment:
Looks like some of the text in this file is hard wrapping at 80 characters. We should unwrap per the doc guidelines.

Reply:
Same problem in several other install files, too. I will fix!